resources/templates/md/pages/user-guide.md

```diff
@@ -826,6 +844,15 @@ curl -s 'https://public.yetibot.com/graphql' \
 
 ```
 
+You can configure the positive and negative Emojis used in commands and responses, overriding the default 🌈 and ⛈.  Have a look at `config/config.sample.edn`
```

Comment:
Can we link to `config/config.sample.edn`?

https://github.com/yetibot/yetibot.core/blob/master/config/config.sample.edn#L70